releases/development/master/extra/install-chef-suse.sh

```diff
@@ -797,6 +822,7 @@ fi
 # in cookbook dependencies)
 #
 for i in crowbar deployer dns ipmi logging network ntp provisioner pacemaker \
+         suse-enterprise-storage \
```

Comment:
This will break in Cloud as this barclamp is not available; I guess we need to make the list of barclamps to install depend on the product?